jsondb.js

```diff
@@ -116,6 +116,60 @@ export const tryPost = ({ ...props }) => {
   return { status: 400 };
 };
 
+export const tryPut = ({ ...props }) => {
+  if (props.method !== "PUT") {
+    return props;
+  }
+  const { lensPath, json, body } = props;
+  const path = [...lensPath];
+  const view = U.view(U.lensPath(path))(json);
+  const parentPath = path.slice(0, -1);
+  const parentView = U.view(U.lensPath(parentPath))(json);
+  if (!view && parentView) {
+    return U.isArray(parentView)
+      ? {
+          data: U.setLens({
+            path: parentPath,
+            content: parentView.concat(U.isArray(body) ? [...body] : body),
+            obj: json,
+          }),
+        }
+      : { status: 400 };
```

Comment:
There should be now some function which deals with statuses, like `getStatus(status = 400, description = null)`, it's been too many times repeated by now.